consensus/src/epoch_manager.rs

```diff
@@ -507,6 +642,28 @@ impl EpochManager {
                     "process_sync_info",
                     p.process_sync_info_msg(*sync_info, peer_id).await
                 ),
+                VerifiedEvent::CommitVote(cv) => {
+                    if let Some(sender) = &self.sender_commit_msg {
```

Comment:
remove the field to commit_phase_tx, it also should receive VerifiedEvent instead of ConsensusMsg

Reply:
resolved